frontend/www/js/omegaup/components/problem/Settings.vue

```diff
@@ -38,15 +38,14 @@
           T.problemEditFormWallTimeLimit
         }}</label>
         <input
+          v-model="localOverallWallTimeLimit"
```

Comment:
Just for convention:
```suggestion
          v-model="currentOverallWallTimeLimit"
```
We commonly use the prefix `current` when we are going to modify the value of a property.